test/runtests.py

```diff
@@ -346,6 +391,95 @@ def on_delete(queue):
                                self.guardian.del_queue_size]
         self.assertTrue(len(queues_to_delete) == 0)
 
+    def test_binding(self):
+        """Test that you can get the bindings for a queue"""
+        self._create_publisher()
+        self._create_consumer_proc(durable=True)
+        self._wait_for_queue()
+        self._wait_for_queue_record()
+        self._terminate_consumer_proc()
```

Comment:
We do this over and over again now in various tests. Could you put this into a function?